### PR TITLE
Update managed-namespaces configmap

### PR DESCRIPTION
**Pull request by tnierman**
Updates `managed-namespaces` configMap by following instructions at https://github.com/openshift/managed-cluster-config/tree/master/deploy/osd-managed-resources#managed-resources-configmap

### CODE REVIEW

File: resources/managed/all-osd-resources.yaml [hunk 1]
```diff
@@ -83,18 +83,12 @@ Resources:
     name: osd-delete-backplane-serviceaccounts
   - namespace: openshift-build-test
     name: sre-build-test
-  - namespace: openshift-cloud-ingress-operator
```
Comment (mrbarge) [inline resources/managed/all-osd-resources.yaml:86]: Is it expected that this and the other service accounts are being removed as managed resources in this PR? It looks like it is removing all the SAs that are listed/deployed via `deploy/osd-serviceaccounts`.

Comment (tnierman) [reply resources/managed/all-osd-resources.yaml]: Interesting, looks like rerunning the script fixed the missing serviceaccounts. Thanks for catching this!

**Review by NautiluX — COMMENTED** [review]
/lgtm

**Comment by cblecker** [comment]
/lgtm cancel

Needs a `make` to regenerate the templates

**Pull request merged**